tools/bumper/component-commands.go

```diff
@@ -0,0 +1,311 @@
+package main
```

Comment:
The name of the file should be components_commands.go

Reply:
I was following Quique's [mention](https://github.com/kubevirt/cluster-network-addons-operator/pull/479#discussion_r459907454) to keep package flat (i.e. nor using packages) in order to avoid using public functions.
What do you think?

Reply:
I'm ok with that, but how is that related to the file name?

Reply:
it's not, my bad